### ISSUE TITLE
How to wrap up my own loss function by python for regression task?

### ISSUE THREAD

**Issue by Oleg67** [catboost/catboost#1060]
class CorrectWin(object):
   def __init__ (self, classes, weights):
       self.classes = classes
       self.weights = weights
   def evaluate(self, approxes, targets, weights):
       # approxes, targets, weights are indexed containers of floats
       # (containers with only __len__ and __getitem__ defined).
       # weights parameter can be None.
       # Returns scalar Loss
       loss_W = np.where((approxes > targets) & (self.classes == 1), np.abs(approxes - targets), 0)
       loss_L = np.where((approxes < targets) & (self.classes == 0), np.abs(approxes - targets), 0)
       loss = (loss_w + loss_L) * weights
       return np.sum(loss)

this is give Catboosterror No metrics specified

**Comment by LyzhinIvan** [comment]
Hi, @Oleg67 ! You can look at this tutorial: https://github.com/catboost/catboost/blob/master/catboost/tutorials/custom_loss/custom_loss_and_metric_tutorial.ipynb
When you specify own loss function, you have to specify eval_metric also (built-in or your own one).

**Comment by camrongodbout** [comment]
> Hi, @Oleg67 ! You can look at this tutorial: https://github.com/catboost/catboost/blob/master/catboost/tutorials/custom_loss/custom_loss_and_metric_tutorial.ipynb
> When you specify own loss function, you have to specify eval_metric also (built-in or your own one).

Thank you this is not super evident in the documentation

**Comment by annaveronika** [comment]
We'll improve the docs, thanks for pointing this out!

**Comment by camrongodbout** [comment]
@annaveronika anyway to help contribute to the documentation?
